lib/tapjoy/ldap/cli/group/remove_user.rb

```diff
@@ -0,0 +1,37 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Remove existing user to existing group
+        class RemoveUser
+          def remove_user
+            confirm unless opts[:force]
+            puts Tapjoy::LDAP::API::Group.update(
+              opts[:group], opts[:username], :delete)
+          end
+
+          private
+          def opts
+            @opts ||= Trollop::options do
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not use `::` for method calls.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2633306024/issues/source?bid=3316970&fileBranchId=3318857#l15)